server/routes/varyingLicences/handlers/caseload.ts

```diff
@@ -79,4 +80,14 @@ export default class CaseloadRoutes {
       search,
     })
   }
+
+  getReviewNeededCasesToTop(a: { licenceStatus: LicenceStatus }, b: { licenceStatus: LicenceStatus }) {
+    if (a.licenceStatus === LicenceStatus.REVIEW_NEEDED && b.licenceStatus !== LicenceStatus.REVIEW_NEEDED) {
+      return -1
+    }
+    if (a.licenceStatus !== LicenceStatus.REVIEW_NEEDED && b.licenceStatus !== LicenceStatus.REVIEW_NEEDED) {
```

Comment:
Had a chat with Nish, I must be misunderstanding how the sort logic works because after his testing and with the unit tests, it shows that the cases are still ordered correctly 🤷